### PR TITLE
Quantum spanning sub queries now passing

### PR DESCRIPTION
**Pull request by gordonguthrie**

### CODE REVIEW

File: src/riak_kv_qry_compiler.erl [hunk 1]
```diff
@@ -461,7 +470,29 @@ simplest_test() ->
 	    ],
     Expected = [Q#riak_sql_v1{is_executable = true,
                               type          = timeseries,
+<<<<<<< HEAD
```
Comment (hazen) [inline src/riak_kv_qry_compiler.erl]: Fix these merge features, @gordonguthrie, and we can talk.

File: src/riak_kv_qry_worker.erl [hunk 2]
```diff
@@ -64,13 +64,14 @@
...
-          qid = undefined :: undefined | {node(), non_neg_integer()},
-          n_chunks = 0    :: non_neg_integer(),
-          status = void   :: void | got_chunk | finished,
-          result = []     :: [binary()]
+          name                 :: qry_fsm_name(),
+          ddl                  :: undefined | #ddl_v1{},
+          qry      = none      :: none | #riak_sql_v1{},
+          qid      = undefined :: undefined | {node(), non_neg_integer()},
+          n_chunks = 0         :: non_neg_integer(),
+	  sub_qrys = []        :: [integer()],
```
Comment (javajolt) [inline src/riak_kv_qry_worker.erl]: off-by-one (white space)

**Comment by hazen** [comment]
Tests now pass

**Comment by hazen** [comment]
:+1: 0b0c67d

File: src/riak_kv_qry_worker.erl [hunk 3]
```diff
@@ -231,13 +248,12 @@ handle_side_effects([{run_sub_query, {qry, Q}, {qid, QId}} | T]) ->
     {ok, _PID} = riak_kv_index_fsm_sup:start_index_fsm(node(), [{raw, QId, Me}, [Bucket, none, Q, Timeout, all, undefined, CoverageFn]]),
     handle_side_effects(T);
 handle_side_effects([H | T]) ->
```
Comment (andytill) [inline src/riak_kv_qry_worker.erl]: This clause shouldn't exist, it would be a bug if it fell into this clause.

Comment (gordonguthrie) [inline src/riak_kv_qry_worker.erl]: I agree

**Comment by gordonguthrie** [comment]
+1 999f67e

**Pull request merged**